_scicompannounce/2022-09-10-rhino-gizmo-updates.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Gizmo Maintenance Work 10 Sept. 2022
+---
+
+On 10 September 2022 the Systems Engineering team will be updating the operating system on the file server _silver_ which hosts fast, home directories, and many other file services essential to our computing environment.
+
+While this is a rolling upgrade which doesn't affect many services, the nature of our HPC work has in past resulted in problems as individual nodes in the _silver_ cluster are rebooted.  Thus, we've decided to take a full outage and to use this as an opportunity to perform some much-needed maintenance on those systems.
+
+This document describes the scope of the outage, the updates we're making, and what you can expect in the lead-up to the outage.
+
+## What to Expect
+
+During this maintenance the rhino login nodes and gizmo compute nodes will be unavailable for any work.  Attempts to connect via ssh will fail.  Any processes running on these nodes will be terminated.
+
+As we have done in the past, we have put in a reservation against all cluster nodes starting at midnight of the 10th. Jobs that would not complete before the maintenance will start will be held until after the work is complete.  `squeue` will show the job as pending with reason _ReqNodeNotAvail, Reserved for maintenance_.
```

Comment:
This might require some clarification. If a job is running before the maintenance, it will not continue running afterwards, will it? If so I would expect that it would not produce the expected results. 
I think what you are saying only applies to jobs that have not started yet, right?

Reply:
If a job is running before the maintenance, then the time requested would indicate that it would finish before the maintenance.  If it goes over time, Slurm will kill it (after 72 hours anyway).

I've _never_ been able to do a real good job explaining this.

Reply:
OK, I think I get it. Others probably won't be quite as dim as me....